### ISSUE TITLE
Image doesn't load if loaded from LiveData.observe callback

### ISSUE THREAD

**Issue by DanielStout5** [MkhytarMkhoian/conductor-glide#2]
In my controller's `onCreateView` I'm observing a `LiveData` and in that observer loading an image with Glide. Like so:

```
override fun onCreateView() {
    data.observe(this, Observer { glide.load(it.image).into(imageView) })
}
```

If I replace the observer with just `Glide.with(context).load(it.image).into(imageView)` it works fine.

That `glide` variable is from an implementation of BaseGlideControllerSupport:

```
    override fun getGlide(): GlideRequests = glideSupport.glide

    private val glideSupport by lazy {
        object : BaseGlideControllerSupport<GlideRequests>(this) {
            override fun getGlideRequest(
                lifecycle: ControllerLifecycle,
                requestManagerTreeNode: RequestManagerTreeNode?
            ): GlideRequests {
                return GlideRequests(
                    Glide.get(context.applicationContext),
                    lifecycle,
                    requestManagerTreeNode!!,
                    context.applicationContext
                )
            }
        }
    }
```

My controller extends the `LifecycleController` from the main Conductor project.

**Comment by MkhytarMkhoian** [comment]
Hi
try to use getGlide().load("url") not a Glide.with(context)

**Comment by DanielStout5** [comment]
Yes, getGlide was what I was originally calling. It doesn't work, but calling the normal Glide.with(applicationContext) does work, so I can only assume the issue must be in your library.

**Comment by MkhytarMkhoian** [comment]
This library has no effect on Glide.with(applicationContext) it's all about Controller lifecycle
I assume the problem  in your LiveData observer

**Comment by MkhytarMkhoian** [comment]
If you calling Glide.with(applicationContext) it is not using BaseGlideControllerSupport

**Comment by DanielStout5** [comment]
Yes, I know. When I call it using BaseGlideControllerSupport, it doesn't work.

I don't think there is an issue in the Live Data observer since it does work with the normal glide call.

**Comment by MkhytarMkhoian** [comment]
In that case, I think the problem in the different lifecycle of Controller and  LiveData observer
